src/include/stir/Scanner.h

```diff
@@ -506,6 +509,7 @@ class Scanner
   shared_ptr<DetectorCoordinateMap> detector_map_sptr;
 
   void set_detector_map( const DetectorCoordinateMap::det_pos_to_coord_type& coord_map );
+  void set_max_FOV_radius();
```

Comment:
Possibly this naming is confusing. `set*` functions normally take a value. We could do that here as well, and if it's negative or so, use the current code. It might get a bit messy though to know if it has been done or not. Probably simpler for now is to rename this private function to  `initialise_max_FOV_radius` and add a `set` function when someone asks for it.

Reply:
I like the "initialise_max_FOV_radius" - will change!

Reply:
done